content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -20,7 +20,7 @@ This page explains how Kubernetes objects are represented in the Kubernetes API,
 * The resources available to those applications
 * The policies around how those applications behave, such as restart policies, upgrades, and fault-tolerance
 
-A Kubernetes object is a "record of intent"--once you create the object, the Kubernetes system will constantly work to ensure that object exists. By creating an object, you're effectively telling the Kubernetes system what you want your cluster's workload to look like; this is your cluster's *desired state*.
+A Kubernetes object is a "record of intent"--once you create the object, the Kubernetes system will constantly work to ensure that the objects it describes exists. By creating an object, you're effectively telling the Kubernetes system what you want your cluster's workload to look like; this is your cluster's *desired state*.
```

Comment:
How about:
```suggestion
Most Kubernetes objects represent some kind of "record of intent"—once you
create the object using the Kubernetes API, your cluster constantly works to ensure
that what you described in that object exists, and is set up the way you defined.
By creating an object, you're effectively telling the Kubernetes system what you
want your cluster's workload to look like.

If you delete an object from Kubernetes' API, you're explicitly saying that the intent
you once had has changed. Depending on exactly what you had previously set up,
the next steps can vary, but the most common action to take after object deletion
is to also delete whatever Kubernetes had set up for you. For example, if you create
a Service object that defines a load balancer, then later delete the Service, Kubernetes
deconfigures that load balancer.
```

?

I'm skipping “desired state” here because it's defined in the next section anyway.